.github/workflows/test-package.yml

```diff
@@ -14,7 +14,6 @@ env:
 
 jobs:
   # Check code formatting and static analysis on a single OS (linux)
```

Comment:
Send sentence with "." ?

Reply:
Done. (I don't know why I always keep forgetting this 🙈)